### PR TITLE
enhancement: make event id copyable

### LINKED ISSUE #5644: [Task]: Make EventId copyable in Proposal Information
### Task description

There doesn't seem to be a way to see the full EventID once it is submitted. It might be worth allowing to copy/paste this. For in the case someone wants to keep a log or something of votes they participated in.
Image

image

### Requirements

TBD

### Creation checklist

- [ ] I have assigned this task to the correct people
- [X] I have added the most appropriate labels
- [X] I have linked the correct milestone and/or project

### PR DESCRIPTION
**Pull request by MarkNerdi**
## Summary

> Please summarize your changes, describing __what__ they are and __why__ they were made.

...

## Changelog

```
- Please write a list of changes
```

## Issue
Closes #5644 

### Platforms

> Please select any platforms where your changes have been tested.

- __Desktop__
	- [ ] MacOS
	- [ ] Linux
	- [ ] Windows
- __Mobile__
	- [ ] iOS
	- [ ] Android

### Instructions

> Please describe the specific instructions, configurations, and/or test cases necessary to __test and verify__ that your changes work as intended.

...

## Checklist

> Please tick all of the following boxes that are relevant to your changes.

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or modified tests that prove my changes work as intended
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have verified that my latest changes pass CI workflows for testing and linting
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by maxwellmattryan — CHANGES_REQUESTED** [review]
Just one minor question

File: packages/shared/components/organisms/ProposalInformation.svelte [hunk 1]
```diff
@@ -6,13 +6,19 @@
...
     import { proposalsState, selectedProposal } from '@contexts/governance/stores'
 
-    const proposalInformation = {
-        countingEnds: formatDate(
-            milestoneToDate($networkStatus.currentMilestone, $selectedProposal.milestones?.ended),
-            DATE_FORMAT
-        ),
-        eventId: truncateString($selectedProposal?.id, 9, 9),
-        nodeUrl: $proposalsState[$activeProfileId]?.[$selectedProposal?.id].nodeUrl,
+    const proposalInformation: { [key in string]: { data: string; isCopyable?: boolean; copyValue?: string } } = {
```
[the remark this answers is not in the document]
Comment (MarkNerdi) [reply packages/shared/components/organisms/ProposalInformation.svelte:9]: i always pushed that behind, so i think now it is necessary

[the remark this answers is not in the document]
Comment (maxwellmattryan) [reply packages/shared/components/organisms/ProposalInformation.svelte:9]: Okay, no problem. Would you mind making a type for that before I merge this? Code looks good and is working as expected :+1:

File: packages/shared/lib/core/utils/interfaces/key-value-box-list.interface.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+export interface IKeyValueBoxList {
+    [key: string]: {
+        data: string
+        isCopyable?: boolean
+        copyValue?: string
+        isTooltipVisible?: boolean
+        alternateKey?: string
```
Comment (maxwellmattryan) [inline packages/shared/lib/core/utils/interfaces/key-value-box-list.interface.ts:7]: Are we missing `tooltipText?: string` here? (from `TokenActivityInformation.svelte`)

**Review by maxwellmattryan — CHANGES_REQUESTED** [review]
Might be missing a property for the type?

**Review by maxwellmattryan — APPROVED** [review]
LGTM :fire:

**Pull request merged**